director/accounts/db_facade.py

```diff
@@ -32,22 +32,22 @@ class AccountFetchResult(typing.NamedTuple):
 
 def fetch_account(
     user: AbstractUser,
-    account_pk: typing.Optional[int] = None,
-    name: typing.Optional[str] = None,
+    account_name: typing.Optional[str] = None,
+    pk: typing.Optional[int] = None,
```

Comment:
I wanted to be consistent with the project shortcut methods, so I've renamed `name` to `account_name`.

Reply:
And `account_pk` to `pk`.